internal/clusterconfig/clusterconfig.go

```diff
@@ -50,43 +80,48 @@ type UpgradeClusterConfigGather struct {
 // FetchClusterConfig collect the current cluster config and write it as json files into data dir:
 func (r *UpgradeClusterConfigGather) FetchClusterConfig(ctx context.Context, ostreeDir string) error {
 	// TODO: Add the following
-	// ssh keys
 	// Other Machine configs?
 	// additionalTrustedCA (from the images.config)
 	// Other Certificates?
 	// Catalog source
 	// ACM registration?
 	r.Log.Info("Fetching cluster configuration")
-	pullSecret := &corev1.Secret{}
-	if err := r.Client.Get(ctx, types.NamespacedName{Name: pullSecretName, Namespace: configNamespace}, pullSecret); err != nil {
+
+	cc := newClusterConfig()
```

Comment:
> EDIT: ugh clusterConfig is the name of the type because it's private so it can't be used as a variable name. I don't like Go. I have no good suggestions here, either rename the type or keep it cc, I give up

Exactly, I'll also try to think it through and update you here if I come up with anything better than what we currently have.

Reply:
I tried to make it a bit clearer by renaming the struct to `clusterConfiguration` and the object used in that scope to `clusterConfig`. WDYT?